app/controllers/scientist_experiments_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ScientistExperimentsController < ApplicationController
+  def destroy
+    scientist = Scientist.find(params[:id])
+    ScientistExperiment.find(ScientistExperiment.where(scientist_id: params[:id]).where(experiment_id: params[:experiment_id])[0].id).destroy
```

Comment:
Use find_by next time! We don't want a where clause in the controller (but this is 1 infraction allowed in the rubric) :)